packages/datadog-plugin-aws-sdk/src/services/cloudwatchlogs.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const BaseService = require('./base')
+
+class CloudwatchLogsService extends BaseService {
+  _addServiceTags (params, operation, response) {
+    const tags = {}
+
+    // cloudwatach log group name
+    if (!params || !params.logGroupName) return tags
+
+    return Object.assign(tags, {
+      'resource.name': `${operation} ${params.logGroupName}`,
+      'aws.cloudwatch_logs.log_group_name': params.logGroupName
```

Comment:
`aws.cloudwatch.logs.log_group_name` would allow to group different CloudWatch product in the same object.

Reply:
makes sense, updated, tests updated as well